clean_metadata.py

```diff
@@ -7,15 +8,24 @@
 import sys
 import json
 
-from bids.grabbids import BIDSLayout
+from bids import BIDSLayout
 
 
-def main(bids_dir):
+def main(bids_dir, sub, sess):
+    '''
+    Removes unnecessary metadata from scan sidecar jsons
+
+    Parameters
+    ----------
+    bids_dir: path to BIDS dataset
+    '''
     layout = BIDSLayout(bids_dir)
-    scans = layout.get(extensions='nii.gz')
+    scans = layout.get(extension='nii.gz', subject=sub, session=sess)
 
+    """
     KEEP_KEYS = [
-        'AnatomicalLandmarkCoordinates', 'AcquisitionDuration', 'CogAtlasID',
+        'AnatomicalLandmarkCoordinates', 'AcquisitionTime',
+        'AcquisitionDuration', 'CogAtlasID',
         'CogPOID', 'CoilCombinationMethod', 'ConversionSoftware',
         'ConversionSoftwareVersion', 'DelayAfterTrigger', 'DelayTime',
         'DeviceSerialNumber', 'DwellTime', 'EchoNumbers', 'EchoTime',
```

Comment:
```suggestion
        'DeviceSerialNumber', 'DwellTime', 'EchoNumbers', 'EchoTime',
        'EchoTime1', 'EchoTime2',
```

Since we'll start cleaning the jsons again at some point.